beacon/endpoints/rest/datasets.py

```diff
@@ -15,9 +15,8 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 class DatasetParameters(RequestParameters):
-    # requested schemas
-    requestedSchemasServiceInfo = SchemasField()
-    requestedSchemasDataset = SchemasField()
+    requestedSchema = SchemaField('beacon-dataset-v2.0.0-draft.2',
```

Comment:
I'm not sure I understand that field.
It has one option.

Reply:
Currently, there is no other schema for datasets, but we may have one.